moveos/moveos-verifier/src/metadata.rs

```diff
@@ -270,6 +273,59 @@ impl<'a> ExtendedChecker<'a> {
 // Entry Function
 
 impl<'a> ExtendedChecker<'a> {
+    fn check_init_module(&mut self, module: &ModuleEnv) {
+        for ref fun in module.get_functions() {
+            if fun.get_identifier().as_ident_str() != INIT_FN_NAME_IDENTIFIER.as_ident_str() {
+                continue;
+            }
+
+            if Visibility::Private != fun.visibility() {
+                self.env
+                    .error(&fun.get_loc(), "module init function should private")
```

Comment:
module init verify is different from other verify. It is expected to find the init method that satisfies the conditions and initialize it, instead of throwing an exception

Reply:
The init function should be verified during execution and compilation

so I understand that any function that does not comply with the init function regulations should have an exception during compilation in order to modify the code